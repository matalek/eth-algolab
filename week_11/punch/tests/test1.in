10
100 78
7615 9
7615 9
7615 9
6404 8
6404 8
6404 8
3545 5
3545 5
3545 5
3545 5
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
6819 5
6819 5
6819 5
6819 5
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
1024 1
8207 6
8207 6
8207 6
8207 6
8207 6
3878 8
3878 8
3878 8
3878 8
4011 9
4011 9
2914 3
2914 3
2914 3
2914 3
2914 3
2914 3
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
723 8
723 8
723 8
723 8
723 8
723 8
723 8
723 8
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
539 8
539 8
539 8
539 8
539 8
539 8
943 10
3645 10
4022 6
5458 3
9617 3
2804 7
4680 7
5857 1
6808 8
4733 1
100 81
1153 3
1153 3
8059 7
8059 7
8059 7
8059 7
8059 7
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
258 2
258 2
258 2
258 2
258 2
258 2
258 2
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6817 1
6817 1
5422 9
5422 9
8146 1
8146 1
8146 1
8146 1
8146 1
4525 9
4525 9
4727 7
4727 7
6278 4
7763 4
7763 4
7763 4
7763 4
7763 4
7763 4
5721 7
3628 8
3628 8
3628 8
3628 8
4614 9
4614 9
4614 9
4614 9
4484 2
4484 2
4484 2
4484 2
4484 2
4484 2
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
7199 7
7199 7
8642 6
8642 6
8642 6
3823 1
3823 1
6445 3
238 1
7299 4
9955 8
4331 10
4079 9
7733 5
1681 1
1966 8
9947 10
9177 9
100 92
5568 73
5568 73
5568 73
5568 73
5568 73
5568 73
9562 74
9562 74
9562 74
9562 74
9562 74
9562 74
7461 61
7461 61
7461 61
7461 61
7461 61
7461 61
8018 91
7543 75
7543 75
5875 56
5875 56
5875 56
5875 56
5875 56
5875 56
9858 99
9858 99
1932 84
1932 84
1932 84
1932 84
6402 52
6402 52
6402 52
6402 52
6402 52
6402 52
6444 83
6444 83
6444 83
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
3363 94
3363 94
3363 94
3363 94
3363 94
3363 94
3286 75
3286 75
3286 75
3286 75
3134 99
3134 99
1919 98
1919 98
1919 98
1919 98
1919 98
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
3993 58
2811 98
2096 100
100 79
5436 80
5436 80
5436 80
5436 80
5436 80
5436 80
5436 80
3088 63
3088 63
3088 63
1006 90
1006 90
1006 90
1006 90
1006 90
2916 74
2916 74
2916 74
2622 56
2622 56
2622 56
2000 87
2000 87
2000 87
2000 87
2000 87
2000 87
6229 86
6229 86
6229 86
6229 86
6545 54
6545 54
6545 54
6545 54
6545 54
6545 54
6545 54
4822 85
6682 78
1159 87
1159 87
5236 61
5236 61
4880 73
4880 73
4880 73
912 61
912 61
912 61
8021 83
8021 83
8021 83
8021 83
1294 60
2901 75
2901 75
2901 75
2901 75
2901 75
2901 75
2901 75
2912 76
2912 76
3694 60
3694 60
3694 60
3694 60
3694 60
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
3708 52
3708 52
3708 52
7679 79
7679 79
7679 79
7679 79
7679 79
4948 77
4948 77
4948 77
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
1333 79
5805 95
213 91
100 55
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
9222 95
7622 87
7622 87
7622 87
6891 60
6891 60
6891 60
7785 50
7785 50
7785 50
7785 50
7785 50
7785 50
745 59
745 59
745 59
745 59
1218 64
1218 64
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7203 69
9221 99
9221 99
6440 79
6440 79
1428 99
1428 99
1428 99
1428 99
1428 99
1428 99
3747 53
3747 53
8287 52
8287 52
8287 52
8287 52
8287 52
8287 52
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
8887 62
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
2675 79
7339 67
1811 53
6379 62
100 88
3367 53
3367 53
3367 53
3367 53
3367 53
3367 53
7682 88
1153 35
1153 35
1153 35
1153 35
1153 35
1153 35
7279 42
7279 42
7279 42
7060 88
7060 88
7060 88
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7108 1
7108 1
7108 1
237 71
237 71
237 71
237 71
1487 56
1487 56
1487 56
1487 56
1487 56
3659 34
3659 34
3659 34
3659 34
8811 67
8811 67
8811 67
8811 67
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
9 40
9 40
9 40
9 40
5373 48
7446 48
5465 48
7664 48
19 15
15 15
16 15
16 15
16 73
19 73
14 73
11 73
2409 13
7509 2
5462 82
8184 93
9263 24
8230 4
8585 6
9593 21
688 70
593 41
1777 54
2949 69
1439 99
2676 97
8494 45
6082 50
4284 13
1111 92
1224 42
8792 25
4757 9
9055 31
2707 86
3795 53
9215 49
5686 100
6226 81
9887 83
4095 63
100 68
9945 47
9945 47
9945 47
9945 47
9945 47
9945 47
1078 44
1078 44
1078 44
1078 44
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
7634 13
7634 13
7634 13
7634 13
7634 13
2918 90
2918 90
2918 90
2918 90
2918 90
2918 90
7083 20
7083 20
7083 20
7083 20
7083 20
7083 20
1814 59
1814 59
1814 59
1814 59
5787 50
5787 50
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
1 8
1 8
1 8
1 8
9543 60
5038 60
6413 60
7565 60
16 10
11 10
14 10
18 10
18 58
16 58
12 58
13 58
2760 82
4349 21
8077 16
2623 91
8368 5
4834 52
1926 43
3086 97
3463 59
8318 93
7310 38
8623 67
3529 90
1892 29
7194 49
1432 1
5867 18
6724 46
1604 13
6344 54
4546 78
9089 49
6393 73
5345 65
6927 100
5531 6
6406 11
6136 48
4930 22
9137 66
5688 3
8935 88
100 87
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 13
1 13
1 13
1 13
1 13
1 17
1 17
1 17
1 17
1 17
1 17
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 13
1 13
1 13
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 2
1 16
1 12
1 5
1 8
1 16
100 52
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 15
1 15
1 15
1 15
1 15
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 6
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 12
1 13
1 13
1 13
1 13
1 3
1 3
1 3
1 3
1 3
1 3
1 3
1 16
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 17
1 9
1 9
1 9
1 9
1 18
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 5
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 19
1 14
1 12
1 12
100 69
1 12
1 12
1 12
1 12
1 12
1 12
1 2
1 2
1 2
1 2
1 2
1 2
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 2
1 2
1 2
1 2
1 2
1 2
1 2
1 11
1 11
1 11
1 11
1 11
1 11
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 5
1 5
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 20
1 9
1 9
1 9
1 9
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 1
