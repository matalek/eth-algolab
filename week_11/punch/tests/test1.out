5390 6
10558 8
560 1
213 1
745 1
20 2
9 4
5 5
3 3
4 4
